src/corecel/io/BuildOutput.cc

```diff
@@ -11,12 +11,12 @@
 #include <utility>
 #include <nlohmann/json.hpp>
 
-#include "celeritas_cmake_strings.h"
 #include "celeritas_config.h"
-#include "celeritas_version.h"
 #include "corecel/Macros.hh"
 
 #include "JsonPimpl.hh"
+#include "celeritas_cmake_strings.hh"
+#include "celeritas_version.hh"
```

Comment:
Please also update the `~/.clang-format` and reformat all these files.

Reply:
What do you want to update exactly in the `.clang-format`? Also, which version of clang-format do we want to support? For example in clang 13, `SortIncludes` [changed](https://releases.llvm.org/13.0.0/tools/clang/docs/ReleaseNotes.html#clang-format) from boolean to enum. There are a few other keys like that (`AllowShortBlocksOnASingleLine` and `Standard`)

Reply:
For `IncludeCategories` I suppose we want to give `"corecel/Version.hh"` a higher priority.